Peterburnett/moodle-mod_observation
# 16 written observation point

Adds observation point as a text input type and view them in table. Also ability to edit/delete/move up & down in table. 

Tests WIP.
Test plan:
- Add using normal data
- Add using wrong/unexpected data
- Odering move up / down works normally
- Delete point with one above and below, ensure ordering is kept intact
- Move point at bottom / top ensure ordering doesn't change

Closes #6 and #16
